### DISCUSSION TITLE
Smart contract ETH withdrawal

### DISCUSSION THREAD

**Discussion by GitBasil** [hashlips-lab/nft-erc721-collection#258] [Q&A] [answered]
Hi,
Thank you guys the tremendous work that you do for the community.
I'm net to the smart contracts, I couldn't find the script for the ETH withdrawl call.
Can you guys show us how to call the withdrawal function please.

**Comment by liarco** [comment, marked as answer]
Hi @basel-ab, there are some useful videos in the README file (you can access it from the home page of this project). The video about this topic is this one: https://youtu.be/zhvTJhBbtnE 😉

**Reply by GitBasil** [reply]
awesome, Thx
